frontend/src/component/releases/ReleasePlanTemplate/TemplateForm/MilestoneList/MilestoneCard/MilestoneCardDragHandle.tsx

```diff
@@ -0,0 +1,54 @@
+import DragIndicator from '@mui/icons-material/DragIndicator';
+import { styled } from '@mui/material';
+import { ScreenReaderOnly } from 'component/common/ScreenReaderOnly/ScreenReaderOnly';
+import type { FC } from 'react';
+
+const DragButton = styled('button')(({ theme }) => ({
+    padding: 0,
+    cursor: 'grab',
+    transition: 'background-color 0.2s ease-in-out',
+    backgroundColor: 'inherit',
+    border: 'none',
+    borderRadius: theme.shape.borderRadiusMedium,
+    color: theme.palette.text.secondary,
+    ':hover, :focus-visible': {
+        outline: 'none',
+        '.draggable-hover-indicator': {
+            background: theme.palette.table.headerHover,
+        },
+    },
+}));
+
+const DraggableContent = styled('span')(({ theme }) => ({
+    paddingTop: theme.spacing(2),
+    paddingInline: theme.spacing(0.5),
+    display: 'block',
+    height: '100%',
+    width: '100%',
+}));
+
+const DraggableHoverIndicator = styled('span')(({ theme }) => ({
+    display: 'block',
+    paddingBlock: theme.spacing(0.75),
+    borderRadius: theme.shape.borderRadiusMedium,
+    '> svg': {
+        verticalAlign: 'bottom',
+    },
+}));
+
+type Props = {
+    dragItemRef: React.RefObject<HTMLElement>;
+};
+
+export const MilestoneCardDragHandle: FC<Props> = ({ dragItemRef }) => {
+    return (
+        <DragButton tabIndex={-1} type='button'>
```

Comment:
These don't support keyboard drag-n-drop (yet?), so I'm taking them out of the flow for now.